content/agent/_index.md

```diff
@@ -3,103 +3,137 @@ title: Agent
+|SmartOS|`svcadm enable datadog`|`n/a`|
+|Source|`sudo ~/.datadog-agent/bin/agent start`|`sudo service datadog-agent start`|
+|Windows|[Consult our dedicated windows doc](/agent/basic_agent_usage/windows)|[Consult our dedicated windows doc](/agent/basic_agent_usage/windows)|
 
-* **The collector**: runs checks on the current machine for whatever [integrations](/integrations) you have and it captures system metrics such as memory and CPU.
+### Stop the agent
 
-* **DogStatsD**: It is a statsd backend server you can send [custom metrics](/getting_started/custom_metrics/) to from an application.
+|Platform|Agent v5 |Agent v6|
+|:--------|:-----|:--------|
+|Linux|`sudo service datadog-agent stop`|`sudo service datadog-agent stop`|
+|MacOS x|`/usr/local/bin/datadog-agent stop` |`launchctl stop com.datadoghq.agent` or systray app  |
+|SmartOS|`svcadm disable datadog`|`n/a`|
+|Source|`sudo ~/.datadog-agent/bin/agent stop`|`sudo service datadog-agent stop`|
+|Windows|[Consult our dedicated windows doc](/agent/basic_agent_usage/windows)|[Consult our dedicated windows doc](/agent/basic_agent_usage/windows)|
+### Restart the agent 
 
-* **The forwarder**: retrieves data from both DogStatsD and the collector and then queues it up to be sent to Datadog.
+|Platform|Agent v5 |Agent v6|
+|:--------|:-----|:--------|
+|Linux|`sudo service datadog-agent restart`|`sudo service datadog-agent restart`|
+|MacOS x|`/usr/local/bin/datadog-agent restart `|_run `stop` then `start`_ or systray app|
+|SmartOS|`svcadm restart datadog`|`sudo service datadog-agent restart`|
+|Source|`sudo ~/.datadog-agent/bin/agent restart`|`n/a`|
+|Windows|[Consult our dedicated windows doc](/agent/basic_agent_usage/windows)|[Consult our dedicated windows doc](/agent/basic_agent_usage/windows)|
 
-This is all controlled by one supervisor process. We keep this separate so you don't have to have the overhead of each application if you don't want to run all parts, although we generally recommend you do.
+## Agent Status and information
 
-<div class="alert alert-info">
-This documentation covers Agent versions 5.0.0 and above.
-</div>
+### Agent Status
 
-## Configuration management tools
+|Platform|Agent v5 |Agent v6|
+|:--------|:-----|:--------|
+|Linux|`sudo service datadog-agent status`|`sudo datadog-agent status`|
+|Docker|`sudo docker exec -it dd-agent /etc/init.d/datadog-agent status`|`n/a`|
+|MacOS x|`datadog-agent status`             | `launchctl list com.datadoghq.agent` or systray app|
+|SmartOS|`svcs datadog`|`n/a`|
+|Source|`sudo ~/.datadog-agent/bin/agent status`|`sudo service datadog-agent status`|
+|Windows|[Consult our dedicated windows doc](/agent/basic_agent_usage/windows/#status-and-information)|[Consult our dedicated windows doc](/agent/basic_agent_usage/windows/#status-and-information)|
 
-Manage the Datadog agent and [integrations](/integrations) using configuration management tools:
+### Agent information
```

Comment:
Lets be consistent in header style. It should either be Title Case or not. Gobstopper's style guide is no title case. Lets just pick one.

Reply:
👌